src/corner_kick/src/utils/ros/index.ts

```diff
@@ -36,28 +36,31 @@ export const stop = () => {
 
 /**
  * Subscribe to ROS Topic
- * @param {string} name - The name of the topic
- * @param {string} messageType - The message type of topic
- * @callback - The callback upon subscribing
+ * @param name - The name of the topic
+ * @param messageType - The message type of topic
+ * @param callback - The callback upon subscribing
+ * @param throttle - The rate at which to limit messages arriving to the visualizer
  */
 export const subscribeToROSTopic = (
     name: string,
     messageType: string,
     callback: (message: ROSLIB.Message) => void,
+    throttle?: number,
 ) => {
     const topic = new Topic({
         messageType,
         name,
         ros,
+        throttle_rate: throttle,
     });
     topic.subscribe(callback);
 };
 
 /**
  * Unsubscribe from ROS Topic
- * @param {string} name - The name of the topic
- * @param {string} messageType - The message type of topic
- * @callback - The callback upon unsubscribing
+ * @param name - The name of the topic
+ * @param messageType - The message type of topic
+ * @param callback - The callback upon unsubscribing
```

Comment:
Just to make sure I understand this line, would it be correct to say:
``` suggestion
 * @param callback - The callback that is called after unsubscribing
```
?

Reply:
False, this is the callback we are unsubscribing. Aka, the callback we no longer want messages to be sent to. Doc clarified.